frontend/src/components/Schema.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Glyphicon } from 'react-bootstrap';
+import './Schema.less';
+
+class SchemaTable extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { expanded: false };
+
```

Comment:
spaces?

Reply:
yes

Reply:
are we applying any special rule/logic in behind?
I saw other quite condense files, so I wonder about the difference in between...

Reply:
I put an empty line between logic blocks:
```
super() - first block

this.state = {} - second

lots of bindings - third
```